golden/clr-angular.d.ts

```diff
@@ -180,6 +180,21 @@ export declare enum ClrAlignment {
     END = 1
 }
 
+export declare enum ClrAriaLivePoliteness {
+    off = "off",
+    polite = "polite",
+    assertive = "assertive"
+}
+
+export declare class ClrAriaLiveService implements OnDestroy {
+    get id(): string;
```

Comment:
Do we need to have `id` in this service? I don't see it serving any purpose.

Ideally, we should expose only announce method from this service.

Reply:
`id` is the only way to know where and what is the id of the aria-container that you are using. And because we don't provide additional methods for cleaning there is no other way to clean it if you needed. 

For example, when you want to have the same message twice the only way will be to remove the content wait for a little and announce it again so the SR will pick the change.

Reply:
We don't handle that case and it's left to people having to handle it find a solution .. 

May be in later iteration this could be improved, but without the ID with the design for multiple instances you won't be able to find the container inside the DOM.

Reply:
This is not a blocker issue. But currently, the service doesn't use to access the element. It will always keep the reference to the element. Currently, it doesn't do anything but kept in the service. And I don't think of any good reason for users to clean it up manually.